service/pipeline.ml

```diff
@@ -7,6 +7,28 @@ module Docker = Current_docker.Default
 module Common = Opam_repo_ci_api.Common
 module Distro = Dockerfile_opam.Distro
 
+(* https://github.com/ocurrent/opam-repo-ci/issues/260 *)
+let host_is_macos =
```

Comment:
How are opam and Dune doing this? I suspect some corner cases can be covered and they would probably have been added in these projects.